tests/test_geoprocessing.py

```diff
@@ -2692,7 +2695,8 @@ def test_convolve_2d_ignore_nodata(self):
 
         # calculate by working on some graph paper
         expected_result = 9*9*.5
-        self.assertAlmostEqual(numpy.sum(target_array), expected_result)
+        self.assertAlmostEqual(numpy.sum(target_array), expected_result,
+                               places=5)

```

Comment:
How are you choosing the decimal places here? There's a 3, 4, and 5. Is there a reason for that that's worth considering? ... Like, I'm worried if there's a numerical error happening.

Reply:
I selected those numbers because they were the maximum values I could select to achieve the same tolerance.  The differences were only observed on tests that used the 2d convolution, and I was observing them in my 64-bit conda environment (on windows).  I'd be happy to dig into this further if you'd like!  I do know that the tests were failing on my machine before I changed anything in the `symbolic` module, so these shouldn't have anything to do with the functional changes in this PR.